webalyzer/collector/static/collect.js

```diff
@@ -1,15 +1,12 @@
-!setTimeout(function() {
-
-  var domain, script = document.querySelector('script[data-webalyzer]');
-  if (script === null || (script !== null && !script.dataset.webalyzer)) {
-    console.warn(
-      "You have not set up your webalyzer collector " +
-      "correctly. It needs to have the domain in the script tag. " +
-      "For example: <script data-webalyzer=\"example.com\">"
-    );
-  } else {
-    domain = script.dataset.webalyzer;
-  }
+var Webalyzer = function(domain, options) {
+  options = options || {};
+  options.doSend = options.doSend || function(html, url) {
+    return true;
+  };
+  options.beforeSendHTML = options.beforeSendHTML || function(html, url) {
+    return html;
+  };
+  options.server = options.server || 'https://webalyzer.dev/collector/';
```

Comment:
Perhaps the default should be the heroku URL? Can you remind me what that was @groovecoder ?